docs/src/fs.rst

```diff
@@ -353,3 +353,48 @@ Helper functions
    any attempts to close it or to use it after closing the fd may lead to malfunction.
 
     .. versionadded:: 1.12.0
+
+File open constants
```

Comment:
Maybe we should document what each flag means, and on which platforms those are supported?

Reply:
I didn't want to duplicate everything again from all the `man` pages...

I figured people using libuv would know, but I can add definitions if you think it's a good idea.

> on which platforms those are supported?

Sure, will do (especially considering how we map `O_DIRECT` etc now on Windows).

Reply:
Done!